content/en/docs/components/spark-operator/api-docs.md

```diff
@@ -0,0 +1,3327 @@
+---
```

Comment:
I think, we decided for now to keep API docs in the Spark Operator repo, since we run auto-gen script to update them.
Maybe we can redirect to this doc from the [kubeflow.org](https://kubeflow.org/) ?

Reply:
WDYT @StefanoFioravanzo ?

Reply:
Let's definitely link to the external API docs, if we have them

Reply:
Changed to use external links.